tests/pytest_framework/src/syslog_ng/syslog_ng_cli.py

```diff
@@ -105,9 +126,9 @@ def stop(self, unexpected_messages=None):
             # wait for stop and check stop result
             if result["exit_code"] != 0:
                 self.__error_handling()
-            if not self.__syslog_ng_ctl.wait_for_control_socket_stopped():
+            if not wait_until_false(lambda s : s.__is_process_running(), self):
```

Comment:
Ok, agreed to change from checking stats to checking process, but lets use the new __wait_for_process_alive()

Reply:
as mentioned above, wait_for_process_alive cannot help here.